src/project-infos.js

```diff
@@ -113,6 +113,29 @@ const getAuthorName = packageJson => {
   return undefined
 }
 
+/**
+ * Returns an object with dynamic badge information if the package exists on NPM.
+ * It returns data for static badge if it does not exist on NPM.
+ *
+ * @param projectName
+ * @param packageJson
+ * @returns {Object{type:string, data:string}}
+ * Type of badge can be "static" or "dynamic". Data can be undefined.
+ */
+const getVersionBadgeInfo = (projectName, packageJson) => {
```

Comment:
I think that returning an object is a bit overkill. The only information we need to get is if project is available on npm or not. We should have a function named `isProjectAvailableOnNpm` that return a boolean. This boolean would be used in the ejs template to tell if we render a static or dynamic badge